verify.py

```diff
@@ -27,9 +34,46 @@ def checkJSONFile(path):
 
 		except FileNotFoundError:
 
-			print("Couldn't find json_verify. You need to install yajl (yet-another-json-lib).", file = sys.stderr)
+			printerr("Couldn't find json_verify. You need to install yajl (yet-another-json-lib).")
 			sys.exit(2)
 
+def checkDuplicateEntries(path):
+
+	# the JSON spec does allow for duplicate dictionary key entries to
+	# appear. And the python json parser acts in some (undefined) way in
+	# this situation, no error is raised.
+	#
+	# therefore hook into the parser to detect duplicate keys and error
+	# out in this case
+
+	errors = []
+
+	def check_duplicates(ordered_pairs):
+
+		d = {}
```

Comment:
minor nit: you could use just a `set` here

Reply:
True. I've changed it.